### PR TITLE
doc/mgr/balancer: document

### PR DESCRIPTION
**Pull request by liewegas**
Not sure how we missed this for luminous!

Fixes: http://tracker.ceph.com/issues/22789

### CODE REVIEW

**Comment by liewegas** [comment]
@dvanders look good to you?

**Comment by dvanders** [comment]
@liewegas lgtm!

File: doc/mgr/balancer.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+No adjustments will be made to the PG distribution if the cluster is
+degraded (e.g., because an OSD has failed and the system has not yet
+healed itself).
+
+When the cluster is healthy, the balancer will throttle its changes
+such that the percentage of PGs that are misplaced (i.e., that need to
+be moved) is below a threshold of (by default) 5%.  The
+``max_misplaced`` threshold can be adjusted with::
+
+  ceph config-key set mgr/balancer/max_misplaced .07   # 7%
```
Comment (xiexingguo) [inline doc/mgr/balancer.rst]: The awkward thing here is that a reboot of the mgr process is required in order to make these settings take effect immediately...

Comment (liewegas) [reply doc/mgr/balancer.rst]: This is a problem with config-key refreshing on the mgr... see https://github.com/ceph/ceph/pull/20458

**Pull request merged**